test/functional/lua/fs_spec.lua

```diff
@@ -503,4 +503,27 @@ describe('vim.fs', function()
       end)
     end
   end)
+
+  describe('relpath()', function()
+    it('works', function()
+      eq(nil, vim.fs.relpath('/var/lib', '/var'))
+      eq(nil, vim.fs.relpath('/var/lib', '/bin'))
```

Comment:
test for relative case, plus some weird inputs?

maybe some whitespace and redundant slashes too...

```suggestion
      eq(nil, vim.fs.relpath('/var/lib', '/bin'))
      eq(nil, vim.fs.relpath('/var/lib', 'bin'))
      eq(nil, vim.fs.relpath('/var/lib', './bin'))
      -- ?
      eq(nil, vim.fs.relpath('/var/lib', '././'))
      -- ?
      eq(nil, vim.fs.relpath('/var/lib', '../'))
      -- ?
      eq(nil, vim.fs.relpath('/var/lib', '/'))
      -- ?
      eq(nil, vim.fs.relpath('/var/lib', '//'))
```